### DISCUSSION TITLE
Train classifier as downstream task on pretrained self-supervised model on custom dataset

### DISCUSSION THREAD

**Discussion by guneetmutreja** [open-mmlab/mmclassification#1578] [Q&A] [answered]
I pretrained my SimCLR model on custom dataset, now I need to train a classifier as a downstream task like we used to do in MMSelfSup. However, I can not see MMClassification folder in downstream tasks in MMPretrain repo (though we have MMDetection and MMSegmentation).

Can we not do that using MMPretrain?

**Comment by Ezra-Yu** [comment, marked as answer]
refer to this note https://mmpretrain.readthedocs.io/en/latest/notes/finetune_custom_dataset.html

**Comment by guneetmutreja** [comment]
Thank you.
